src/Appwrite/Utopia/Response/Filters/V15.php

```diff
@@ -0,0 +1,556 @@
+                case Database::PERMISSION_DELETE:
+                case Database::PERMISSION_WRITE:
+                case Database::PERMISSION_CREATE:
+                    $write[$this->parseRole($permission_value)] = true;
+                    break;
+                case Database::PERMISSION_READ:
+                    $read[$this->parseRole($permission_value)] = true;
+                    break;
+            }
+        }
+
+        $content['$read'] = array_keys($read);
+        $content['$write'] = array_keys($write);
+
+        unset($content['$permissions']);
+
+        return $content;
+    }
+
+    protected function parseRole(string $role)
+    {
+        switch ($role) {
+            case Role::any()->toString():
+                return 'role:all';
+            case Role::users()->toString():
+                return 'role:member';
+            case Role::guests()->toString():
+                return 'role:guest';
+            default:
+                return $role;
+        }
+
+        return $role;
+    }
+
+    protected function parseCollection(array $content)
+    {
+        if (isset($content['documentSecurity'])) {
+            if ($content['documentSecurity']) {
+                $content['permission'] = 'document';
+            } else {
+                $content['permission'] = 'collection';
+            }
+        }
+
+        unset($content['documentSecurity']);
+        $content = $this->parsePermissions($content);
+        $content = $this->parseDatetimeAttributes($content, ['$createdAt', '$updatedAt']);
+        return $content;
+    }
+
+    protected function parsePermissionsCreatedAtUpdatedAt(array $content)
+    {
+        $content = $this->parsePermissions($content);
+        $content = $this->parseDatetimeAttributes($content, ['$createdAt', '$updatedAt']);
+        return $content;
+    }
+
+    private function parseExecution($content)
+    {
+        unset($content['stdout']);
+        $content = $this->parsePermissions($content);
+        $content = $this->parseDatetimeAttributes($content, ['$createdAt', '$updatedAt', 'startTime', 'endTime']);
+        return $content;
+    }
+
+    private function parseCreatedAtUpdatedAt($content)
+    {
+        $content = $this->parseDatetimeAttributes($content, ['$createdAt', '$updatedAt']);
+        return $content;
+    }
+
+    private function parseFunction($content)
+    {
+        if (isset($content['execute'])) {
+            foreach ($content['execute'] as $i => $role) {
+                $content['execute'][$i] = $this->parseRole($role);
+            }
+        }
+
+        if (isset($content['vars'])) {
+            $vars = [];
+            foreach ($content['vars'] as $i => $var) {
+                $vars[$var['key']] = $var['value'];
+            }
+            $content['vars'] = $vars;
+        }
+
+        $content = $this->parseDatetimeAttributes($content, ['$createdAt', '$updatedAt', 'scheduleNext', 'schedulePrevious']);
+        return $content;
```

Comment:
we changed the `status` attribute which would previously be "enabled" or "disabled" to a boolean attribute  `enabled` which can be true or false 
https://github.com/appwrite/appwrite/pull/3798